packages/core/src/domain/configuration/endpointBuilder.spec.ts

```diff
@@ -36,7 +37,47 @@ describe('endpointBuilder', () => {
     })
   })
 
-  describe('proxyUrl', () => {
+  describe('proxy configuration', () => {
+    it('should replace the intake endpoint by the proxy and set the intake path and parameters in the attribute ddforward', () => {
+      expect(
+        createEndpointBuilder({ ...initConfiguration, proxy: 'https://proxy.io/path' }, 'rum', []).build('xhr')
+      ).toMatch(
+        `https://proxy.io/path\\?ddforward=${encodeURIComponent(
+          `/api/v2/rum?ddsource=(.*)&ddtags=(.*)&dd-api-key=${clientToken}` +
+            '&dd-evp-origin-version=(.*)&dd-evp-origin=browser&dd-request-id=(.*)&batch_time=(.*)'
+        )}`
+      )
+    })
+
+    it('normalizes the proxy url', () => {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏any specific motivation to introduce this behaviour?

Reply:
I didn't introduce it, it was already there, but not tested in this spec. This is used to prevent intake requests from being considered as a RUM resource: because PerformanceTiming resources urls are absolute, we need to make the proxy url also absolute to compare them.

Reply:
oh, indeed I missed that